### PR TITLE
Finished with hockey leaders. Also some random stuff

### PR DESCRIPTION
**Pull request by blakeparkinson**

### CODE REVIEW

File: models/players.js [hunk 1]
```diff
@@ -568,6 +570,24 @@ var insertTopScorers= function (data){
   
 }
 
+var abbreviationHelper = function(league, abbreviation){
```
Comment (kjgross) [inline models/players.js]: Can you use the existing fxn in top_scores for this?

**Comment by kjgross** [comment]
nhl leaders aren't in mongo so I didn't test, but code looks gooooood. :hamster:

File: models/players.js (hunk 1 above)
Comment (blakeparkinson) [inline models/players.js]: i didnt realize there were 2, so i moved the function over to the players model and referenced it there

**Comment by kjgross** [comment]
:ship:

**Pull request merged**